docs/api-reference/next/image.md

```diff
@@ -144,30 +116,32 @@ const MyImage = (props) => {
 
 ### sizes
 
-A string mapping media queries to device sizes. Defaults to `100vw`.
+A string that provides information about how wide the image will be at different breakpoints. Defaults to `100vw` (the full width of the screen).
+
+`sizes` is important for performance when using `layout="responsive"` or `layout="fill"` with images that take up less than the full viewport width.
 
-We recommend setting `sizes` when using `layout="responsive"` or `layout="fill"` and your image will **not** be the same width as the viewport.
+If you are using `layout="fill"` and the image will always be less than half the viewport width, include `sizes="50vw"`. Without `sizes`, the image will be sent at twice the necessary resolution, decreasing performance.
 
 [Learn more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes).
 
 ### quality
 
-The quality of the optimized image, an integer between `1` and `100` where `100`
-is the best quality. Defaults to `75`.
+The quality of the optimized image, an integer between `1` and `100` where `100` is the best quality. Defaults to `75`.
 
 ### priority
 
 When true, the image will be considered high priority and
-[preload](https://web.dev/preload-responsive-images/).
+[preload](https://web.dev/preload-responsive-images/). Lazy loading is automatically disabled for images using `priority`.
 
-Should only be used when the image is visible above the fold. Defaults to
-`false`.
+You should use the `priority` attribute on any image which you suspect will be the [Largest Contentful Paint (LCP) element](https://nextjs.org/learn/seo/web-performance/lcp). It may be appropriate to have multiple priority images, as different images may be the LCP element for different viewport sizes.
+
+Should only be used when the image is visible above the fold. Defaults to `false`.
 
 ### placeholder
 
-A placeholder to use while the image is loading, possible values are `blur` or `empty`. Defaults to `empty`.
+A placeholder to use while the image is loading. Possible values are `blur` or `empty`. Defaults to `empty`.
 
-When `blur`, the [`blurDataURL`](#blurdataurl) property will be used as the placeholder. If `src` is an object from a static import and the imported image is jpg, png, or webp, then `blurDataURL` will automatically be populated.
+When set to `blur`, the [`blurDataURL`](#blurdataurl) property will be used as the placeholder. If `src` is an object from a [static import](#local-images) and the imported image is `.jpg`, `.png`, or `.webp`, then `blurDataURL` will be automatically populated.
```

Comment:
This is the only usage of "set to" in this document, so lets remove it 

```suggestion
When `blur`, the [`blurDataURL`](#blurdataurl) property will be used as the placeholder. If `src` is an object from a [static import](#local-images) and the imported image is `.jpg`, `.png`, or `.webp`, then `blurDataURL` will be automatically populated.
```

Also note we have the "When empty" sentence below (we could move that up above if it will read better that way)